components/settings/Account/Usage.tsx

```diff
@@ -0,0 +1,63 @@
+import { Account, UsageByService } from "@3rdweb-sdk/react/hooks/useApi";
+import { SimpleGrid, Spinner, VStack } from "@chakra-ui/react";
+import { UsageCard } from "./UsageCard";
+import { useMemo } from "react";
+import numeral from "numeral";
+import { Heading } from "tw-components";
+import { format } from "date-fns";
+
+interface UsageProps {
+  account: Account;
+  usage: UsageByService | undefined;
+  usageLoading: boolean;
+}
+
+export const Usage: React.FC<UsageProps> = ({
+  account,
+  usage,
+  usageLoading,
+}) => {
+  const bundlerTotal = useMemo(() => {
+    if (!usage?.bundler || usage.bundler.length === 0) {
+      return <i>No usage in this period</i>;
+    }
+    const total = usage.bundler
+      .map((u) => u.sumTransactionFeeUsd)
+      .reduce((acc, num) => acc + num, 0);
+
+    return numeral(total).format("($ 0.00 a)");
```

Comment:
Rather than installing a library, can we just [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat)?

```typescript
console.log(new Intl.NumberFormat(undefined, { style: "currency", currency: "USD" }).format("12345.67"));
// $12,345.67 in my locale
```